package.json

```diff
@@ -8,16 +8,16 @@
     "axios": "0.18.0",
     "csstips": "0.3.0",
     "csx": "9.0.0",
-    "cytoscape": "3.2.14",
+    "cytoscape": "3.2.16",
     "cytoscape-canvas": "3.0.1",
     "cytoscape-cola": "2.2.3",
     "cytoscape-cose-bilkent": "4.0.0",
     "cytoscape-dagre": "2.2.1",
-    "cytoscape-panzoom": "2.5.2",
-    "cytoscape-popper": "1.0.1",
+    "cytoscape-panzoom": "2.5.3",
```

Comment:
We don't use this anymore, forgot to remove when removing that widget. Could you do that now or should i do that in a separate PR?

Reply:
Stopped using that since https://github.com/kiali/kiali-ui/pull/631

Reply:
sure. I'll just remove it.